### PR TITLE
wallet: added prompt to dogecoin_wallet_load

### PR DESCRIPTION
**Pull request by edtubbs**
Added prompts for wallet and header database files and moved `fcntl` to `dogecoin_spv_client_discover_peers` for delayed non-blocking stdin.

### CODE REVIEW

**Review by xanimo — APPROVED** [review]
ACK. tested on #188 all memory has been freed

File: doc/tools.md
Comment (michilumin) [inline doc/tools.md:1]: seed to master key function == very good.

File: src/cli/such.c
Comment (michilumin) [inline src/cli/such.c:1]: 👍 ❤️ 👍

File: src/spv.c
Comment (michilumin) [inline src/spv.c:1]: stdin non blocking to accept commands during display for win32, v. good thanks for remembering

**Review by michilumin — APPROVED** [review]
really good and down the right path to user friendliness and all that jazz.

**Pull request merged**